diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -459,6 +502,71 @@ namespace diff_drive_controller{
       left_wheel_joints_[i].setCommand(vel_left);
       right_wheel_joints_[i].setCommand(vel_right);
     }
+
+    // Publish wheel data:
```

Comment:
If the big section below is contained within a function, we may not need this comment at all